### ISSUE TITLE
imiplement display from opal

### ISSUE THREAD

**Issue by johannes-titz** [shevandrin/qti#151]
display is an option to show all items of a section on one page or separate pages.

**Comment by shevandrin** [comment]
it is already implemented, it controls by submission_mode parameter and slot in AssessmentTest, AssessmentTestOpal, test(), test4opal()

**Comment by johannes-titz** [comment]
Okay, so submission mode is the name of the parameter in the qti standard; in opal it is doing two things: showing all exercises of one section on one page and changing the submission mode. This should be added in the test4opal function and maybe in the pkgdown docs.

**Comment by johannes-titz** [comment]
submission_mode is not ideal as a name; maybe display_mode?

**Comment by shevandrin** [comment]
what we have so far.
In Opal for tests there is such option:

<img width="450" height="69" alt="Image" src="https://github.com/user-attachments/assets/29b93124-044f-45e8-80c3-fb1eaabe81a8" />
this option controls attribute "submission_mode" in xml file, where:
submission_mode = individual gives:

<img width="646" height="448" alt="Image" src="https://github.com/user-attachments/assets/255a37d0-282b-4470-8d41-0c5927f1575e" />
submission_mode=simultaneous gives:

<img width="1022" height="458" alt="Image" src="https://github.com/user-attachments/assets/3fc5b0fc-5f0c-4ae8-b3f7-c9abf59793a4" />
(all items from section are displayed altogether and **(!!!) one submit button**)

This is how this parameter is presented in rqti docs:

```
submission_mode
A character value, optional, determining when the candidate's responses are submitted for response processing. Possible values:

"individual" - submit candidates' responses on an item-by-item basis. This is used by default.

"simultaneous" - candidates' responses are submitted all together by the end of the test.
```

in qti 2.1 submission mode is defined:
```
Enumeration: submissionMode

individual

simultaneous

The submission mode determines when the candidate's responses are submitted for response processing. A [testPart](https://www.imsglobal.org/question/qtiv2p1/imsqti_infov2p1.html#element10489) in individual mode requires the candidate to submit their responses on an item-by-item basis. In simultaneous mode the candidate's responses are all submitted together at the end of the testPart.

The choice of submission mode determines the states through which each item's session can pass during the test. In simultaneous mode, response processing cannot take place until the testPart is complete so each item session passes between the interacting and suspended states only. By definition the candidate can take one and only one attempt at each item and feedback cannot be seen during the test. Whether or not the candidate can return to review their responses and/or any item-level feedback after the test, is outside the scope of this specification. Simultaneous mode is typical of paper-based tests.
```

According to this now rqti provides users with expected behaviour, so I would keep "submission_mode" as name of this slot
how do you see it now @johannes-titz ?

**Comment by johannes-titz** [comment]
I agree that `submission_mode` should remain the parameter name because it maps directly to QTI’s `submissionMode`. However, two concepts should be distinguished:

- QTI: when responses are submitted for processing.
- OPAL: whether items are displayed separately or together.

OPAL couples these behaviours, but QTI does not define pagination. Therefore, `display_mode` would be misleading. The documentation should instead mention OPAL’s display behaviour and replace “end of the test” with “end of the test part.”
